assets/js/util/errors.js

```diff
@@ -85,3 +85,21 @@ export function isInsufficientPermissionsError( error ) {
 export function isAuthError( error ) {
 	return !! error?.data?.reconnectURL;
 }
+
+/**
+ * Checks if the given error can be retried.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} error The error object to check.
+ * @return {boolean} TRUE if the error is retryable, otherwise FALSE.
+ */
+export function isErrorRetryable( error ) {
+	return (
+		!! error?.selectorData?.storeName &&
+		error.selectorData.name === 'getReport' &&
```

Comment:
Just a note to mention that if we _do_ continue with this issue, this check for the selector name being `getReport` looks like it should be removed and only checked for the `ReportError` component... But, let's hold off and see if we are actually continuing with the issue or not.

Reply:
Hey @makiost, as we are indeed continuing with this issue, this point should be addressed :point_up: :)